docs/blog/newsletter-march-2023.rst

```diff
@@ -0,0 +1,110 @@
+Semantic Linefeeds
+------------------
+
+If you write the docs in a markup language such as Markdown, AsciiDoc, or reStructured Text, you might have encountered semantic linefeeds (also known as `semantic line breaks <https://sembr.org/>`__). Semantic linefeeds mean that you add a line break after each sentence or even clause in your source file, not just after paragraphs. These shorter lines make the source files easier to read and edit (and the pages are still rendered in paragraphs when published).
+
+Some documentarians dislike the choppy, random look that semantic linefeeds can produce. Others like the technique because it helps with docs-as-code issues such as keeping file comparisons ("diffs") neat and legible and because it helps break down text for translation by line.
+
+Among those who do use semantic linefeeds, the most common form is a variation called ventilated prose: one sentence per line. Adding a line break after every sentence feels somewhat natural and makes it easier to compare sentence lengths. Ventilated prose also makes it more obvious when you have long sentences that could be rewritten. 
+
+-------------------------
+Most interesting projects
+-------------------------
+
+As the role of documentarians has become increasingly crucial in today's fast-paced technological world, they have been tasked with numerous challenging projects. From documenting complex software systems to creating user manuals for cutting-edge devices, documentarians have a lot on their proverbial plates. However, among all these projects, some always stand out as particularly intriguing or memorable. For example, some recall documenting museum software to help guide staff in recording details about everything they collect, such as art, clothing, and dinosaur skeletons.
+
+Others mentioned having a localization voiceover gig to demonstrate using a chainsaw, working on contracts for a company that built and sold childcare facilities, writing instructions for a reading tracking software company, authoring an operations manual for a robotic systems defense contractor, and creating content for a bespoke application for the oil and gas industry. From the variety of examples, it can be said that documentarians are tasked with complex and exciting projects, even if it's just helping a parent write and edit books on sleight-of-hand magic.
+
+The most interesting projects that documentarians have worked on showcase the diverse range of topics and industries they can adapt to and the complexity of the information they need to communicate. As technology continues to evolve and shape our world, documentarians will remain a crucial part of the process, ensuring that information is shared effectively and efficiently with a broad audience. It's clear that it's already a fascinating and challenging field, and it’s expected that more innovative and interesting projects will emerge in the future.
+
+---------------------
+AI for documentarians
+---------------------
```

Comment:
It's super not clear to me whether we're talking about docs for AI or AI for docs.